CondFormats/SiPixelObjects/BuildFile.xml

```diff
@@ -12,6 +14,10 @@
 <use name="FWCore/MessageLogger"/>
 <use name="Geometry/CommonDetUnit"/>
 <use name="Geometry/CommonTopologies"/>
+<use name="HeterogeneousCore/AlpakaCore"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
+<use name="HeterogeneousCore/CUDACore"/>
+<flags ALPAKA_BACKENDS="1"/>
```

Comment:
```suggestion
<flags ALPAKA_BACKENDS="cuda rocm"/>
```

Reply:
sure, I will look in to it

Reply:
Just to remind on the previous discussion on this topic https://github.com/cms-sw/cmssw/pull/40792#discussion_r1109350161 (and an issue https://github.com/cms-sw/cmssw-config/issues/93)

Reply:
Right, I keep forgetting about that.
OK, let's keep it like this for now.